### DISCUSSION TITLE
Welcome to CDK Discussions!

### DISCUSSION THREAD

**Discussion by Xyntax** [cdk-team/CDK#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by saravananidp** [comment]
How can hack someones Android Mobile Phones ?
I need to hack a mobile whatsapp , SMS, Contacts.....How?

**Reply by Xyntax** [reply]
well, that's not the topic here, maybe you can find some ideas in https://github.com/vaib25vicky/awesome-mobile-security

**Comment by wmliang** [comment]
this seems suitable for CDK
https://unit42.paloaltonetworks.com/windows-server-containers-vulnerabilities/

**Reply by neargle** [reply]
Wow, this is Windows Server Containers, we haven't followed up on this technology yet, but this one is interesting, I think it's one of our future TODOs
